iOS/DuckDuckGo/Subscription/UserScripts/SubscriptionPagesUseSubscriptionFeature.swift

```diff
@@ -350,17 +362,24 @@ final class DefaultSubscriptionPagesUseSubscriptionFeature: SubscriptionPagesUse
 
         setTransactionStatus(.polling)
 
+        var subscriptionParameters: [String: String]?
```

Comment:
This is the same as `freeTrialParameters` only renamed to generalize it a bit more.